state.go

```diff
@@ -13,4 +13,6 @@ type State struct {
 	Pid int `json:"pid"`
 	// BundlePath is the path to the container's bundle directory.
 	BundlePath string `json:"bundlePath"`
+	// Annotations holds runtime-specified information that is not structured by this specification.
```

Comment:
But what prevents users from adding their own metadata here? At the Spec level, why do we have to differentiate between runtimes and other users?